civis/resources/_resources.py

```diff
@@ -404,9 +405,25 @@ def parse_path(path, operations, api_version, resources):
 
 
 def parse_swagger(swagger, api_version, resources):
-    """ Parse a swagger specifiction into a dictionary of classes
+    """ Dynamically create classes to interface with the Civis API.
+
+    Parse an OpenAPI (Swagger) specification into a dictionary of classes
     where each class represents an endpoint resource and contains
     methods to make http requests on that resource.
+
+    Parameters
+    ----------
+    api_spec : OrderedDict
```

Comment:
This should be "swagger" instead of "api_spec", no? Although I think that `api_spec` is a better parameter name, so it would be preferable to change the code to match the docs.